### PR TITLE
Custom json config server & new CLI executable

### PR DESCRIPTION
**Pull request by n1k0**
This introduces a new `kinto-admin` executable for serving the Web UI and consuming a JSON configuration file when used as an external dependency:

```
$ kinto-admin -c myconfig.json
```

This also revamps the build and publishing procedures. We now have to version the dist files as one should be able to serve the admin UI with no compile operation when the package is used as a project dependency.

Typical usage, eg. for the AMO blocklist UI:

```
$ npm install kinto-admin --save-dev
$ kinto-admin serve -p 3000 -c amo-blocklist.json
$ kinto-admin build -c amo-blocklist.json -d static-build
```

### CODE REVIEW

**Comment by n1k0** [comment]
This is now ready for review. r=? @natim @leplatrem @almet

File: README.md [hunk 1]
```diff
@@ -2,41 +2,96 @@
...
+A Web admin UI to manage [Kinto](https://kinto.readthedocs.org/) collections.
+[Give it a go](http://kinto.github.io/kinto-admin/)!
 
 ## Installation
 
 NodeJS v4+ and npm 2.14+ should be instaled and available on your machine.
 
 ```bash
-$ npm install
+$ npm install kinto-admin --save
```
Comment (almet) [inline README.md]: Do you really want to save it locally?

File: README.md [hunk 2]
```diff
@@ -2,41 +2,96 @@
...
+        ...
+      }
+    }
+  },
+  // Another collection to manage
+  "articles": {
+    // ...
+  }
+}
 ```
```
Comment (almet) [inline README.md]: I found that writing json schema wasn't really simple to generate. But, the good news is that there is a generator for them, that you can find at http://jsonschema.net/#/. Maybe it's worth mentioning we can generate the schema there?

File: README.md (hunk 1 above)
Comment (n1k0) [inline README.md]: Well if it's a dependency of your project, yes?

File: bin/kinto-admin [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+    fs.copySync(config, path.join(destination, "config.json"));
+  } catch(err) {
+    console.error(err);
+    return process.exit(1);
+  }
+}
+
+function baseCommand(yargs) {
+  return yargs
+  .help("h")
```
Comment (almet) [inline bin/kinto-admin]: nit: indent these?

File: bin/kinto-admin [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+  .alias("h", "help")
+  .option("c", {
+    alias: "config",
+    describe: "Path to the Kinto admin JSON configuration file.",
+  })
+  .demand(["config"])
+  .check(function(argv) {
+    var customConfigPath;
+    if (argv.config) {
+      if (argv.config[0] === "/") {
```
Comment (almet) [inline bin/kinto-admin]: nit: add a comment "extend relative paths to absolute ones"

File: package.json [hunk 5]
```diff
@@ -3,25 +3,32 @@
   "version": "0.3.0",
   "description": "Kinto Web Administration Console in React.js",
   "scripts": {
-    "build": "NODE_ENV=production webpack --optimize-minimize --config webpack.prod.js && cp index.prod.html build/index.html && cp config/config.json build/",
+    "dist": "NODE_ENV=production webpack --optimize-minimize --config webpack.prod.js && cp index.prod.html dist/index.html",
     "lint": "eslint scripts test",
-    "publish": "npm run build && gh-pages --dist build/",
+    "publish": "rimraf build && npm run dist && bin/kinto-admin build -c config/config.json -d build && gh-pages --dist build/",
```
Comment (almet) [inline package.json]: I don't see rimraf in the dependencies

Comment (almet) [inline package.json]: nvm, it's there, in the devDeps.

File: README.md (hunk 1 above)
Comment (almet) [inline README.md]: Yes, of course :-) I was really more thinking about kinto-admin as a standalone thing, not as something that would be a dependency of your project (a lib). So maybe it's worth exchanging to have the "install globally" first, and then the "install as a dependency of your project" in the second place?

**Comment by almet** [comment]
Perfect! r+wc

**Comment by almet** [comment]
Great work man!

**Comment by Natim** [comment]
So no more webpack, everything is handled in kinto-admin. That's great :+1:

**Pull request merged**